packages/app/universal-ts-utils/README.md

```diff
@@ -4,3 +4,16 @@ Isomorphic general-purpose TS utils
 
 ## Overview
 
+
+### String array equality
+
+This package also provides a function to compare string arrays:
```

Comment:
can you clarify the rules for comparison? "same elements in same order, compared by strict equality"